index.bs

```diff
@@ -1465,6 +1465,8 @@ interface XRInputSource {
 };
 </pre>
 
+Note: The {{XRInputSource}} interface is also extended by the <a href="https://immersive-web.github.io/webxr-gamepads-module/">WebXR Gamepads Module</a>
```

Comment:
bikeshed lets you do ``[[!webxr-gamepads-1]]`` or ``[[!webxr-gamepads-1|WebXR Gamepads Module]]`

Reply:
I think this makes it work better in the references section

Reply:
Ah, good to know! It appears that the gamepads module (who's identifier should be `webxr-gamepads-module-1`, FWIW) isn't known to bikeshed yet, though.

Reply:
I'm seeing it in my local bikeshed data, going to poke at this a bit

Reply:
looks like they need to be in biblio-keys.json

Reply:
Yeah so it seems like bikeshed is aware of the spec but not aware _enough_ to generate proper bibliography entries. Pinged Tab on the issue, they'll probably know why

Reply:
For now we should merge this